src/main/java/com/msalvatore/graphqlplayground/mutation/VehicleMutation.java

```diff
@@ -14,7 +16,7 @@ public VehicleMutation(VehicleService vehicleService) {
         this.vehicleService = vehicleService;
     }
 
-    public Vehicle createVehicle(String type, String modelCode, String brandName, String launchDate) {
+    public Vehicle createVehicle(String type, String modelCode, String brandName, LocalDate launchDate) {
```

Comment:
This is the benefit here. The mutation layer now accepts a `LocalDate` and we don't have to worry about parsing the date explicitly.